docker-compose.yaml

```diff
@@ -5,21 +5,22 @@ services:
     build:
       context: ./docker/php
       dockerfile: Dockerfile
-    restart: 'always'
+    restart: 'no'
     depends_on:
       - mariadb
     volumes:
       - .:/var/www/html/
       - ./trust:/var/www/trust
       - ./docker/php/conf.d/xdebug.ini:/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
       - ./docker/php/conf.d/error_reporting.ini:/usr/local/etc/php/conf.d/error_reporting.ini
+      - ./docker/php/conf.d/memory_limit.ini:/usr/local/etc/php/conf.d/memory_limit.ini
```

Comment:
So my only comment is whether it makes sense to have lots of little .ini files or one big one. Seems to me it depends on the number of customizations we're making? If only a few, why not have just one that is our set of minor changes. If many, then perhaps you want to pick and choose under some circumstances and it's easier to edit which collections are included vs editing the .ini directly? This is really a style question probably? Not sure if there's a significant technical reason to suggest one direction or the other.

Reply:
Good point. There's no technical reason for one over the other so I have no issue with consolidating these into a single ini for our overrides.